doc/src/explanation/gotchas.rst

```diff
@@ -377,7 +378,7 @@ you don't have to worry about this problem:
     >>> S("22/7")
     22/7
 
-Also, if you do not use :command:`isympy`, you could use ``from
+Also, if you use Python 2 and do not use :command:`isympy`, you could use ``from
```

Comment:
We should just remove all mentions of this. No one uses Python 2 any more.

Reply:
I agree with that, but IMO it deserves a separate PR, though.
Anyway, [the documentation](https://docs.sympy.org/latest/install.html) says:
> SymPy officially supports Python 3.8, 3.9, 3.10, and PyPy.